app/src/test/java/com/gatchimath/totask/ExampleUnitTest.java

```diff
@@ -2,14 +2,14 @@
 
 import org.junit.Test;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;

```

Comment:
This was auto-generated by IntelliJ. I've reverted files that this PR should not directly affect.